dowhy/api/causal_data_frame.py

```diff
@@ -92,12 +92,14 @@ def do(self, x, method='weighting', num_cores=1, variable_types={}, outcome=None
         if not bool(variable_types): #check if the variables dictionary is empty
             variable_types = dict(self._obj.dtypes) #Convert the series containing data types to a dictionary
             for key in variable_types.keys():
-                variable_types[key] = self.convert_to_custom_type(variable_types[key]) #Obtain the custom type corrosponding to each data type
+                print(variable_types[key])
```

Comment:
Is the print command necessary here? If not, I suggest removing it or logging it in logger.DEBUG

Reply:
Nah, the print is not necessary, I am taking care of data types that are not supported by throwing an exception.